.circleci/config.yml

```diff
@@ -564,22 +564,16 @@ jobs:
       - run:
           name: Check for up-to-date LICENSE.md
           command: |
-            COREMAPS_VERSION=$(jq -r .MapboxCoreMaps scripts/release/packager/versions.json)
             export GITHUB_TOKEN=$(mbx-ci github reader token)
-            DEPENDENCY_CONTENT=$(gh api -H "Accept: application/vnd.github+json" "/repos/mapbox/mapbox-gl-native-internal/contents/LICENSE-iOS.md?ref=maps-v$COREMAPS_VERSION" --jq ".content" | base64 --decode)
+            EXPECTED_LICENSE=$(scripts/license/generate-license.sh)
 
-            # Check if length of dependency content is 0
-            if [ ${#DEPENDENCY_CONTENT} -eq 0 ]; then
-              echo "LICENSE-iOS.md is empty"
-              exit 1
-            fi
-
-            if [[ "$(cat LICENSE.md)" == *"$DEPENDENCY_CONTENT"* ]];
+            if [[ "$(cat LICENSE.md)" == "$EXPECTED_LICENSE" ]];
```

Comment:
It might be a bit more useful to put replace stored license with generated value then check the diff. And print that difference in case of any

Reply:
I though about doing that, but decided to go with more verbose approach and dump the expected license so one could use it to update the license file and make the check happy.